packages/legend-server-depot/src/DepotServerClient.ts

```diff
@@ -236,23 +256,13 @@ export class DepotServerClient extends AbstractServerClient {
     versionId: string,
   ): Promise<Map<string, Entity[]>> {
     const dependencyEntitiesIndex = new Map<string, Entity[]>();
-    let dependencies: PlainObject<ProjectVersionEntities>[] = [];
-    if (versionId === SNAPSHOT_VERSION_ALIAS) {
-      dependencies = await this.getLatestRevisionDependencyEntities(
-        project.groupId,
-        project.artifactId,
-        true,
-        false,
-      );
-    } else {
-      dependencies = await this.getDependencyEntities(
-        project.groupId,
-        project.artifactId,
-        versionId === LATEST_VERSION_ALIAS ? project.latestVersion : versionId,
-        true,
-        false,
-      );
-    }
+    const dependencies = await this.getDependencyEntities(
```

Comment:
can you check if we can remove `getLatestRevisionDependencyEntities()` and remove that method as well? It's such a confusing one 😅